AlfrescoApp/View Controllers/Accounts View Controllers/New Account/AccountInfoDetailsViewController.m

```diff
@@ -381,16 +392,21 @@ - (BOOL)validateAccountFieldsValuesForServer
         
         NSRange hostnameRange = [hostname rangeOfString:@"^[a-zA-Z0-9_\\-\\.]+$" options:NSRegularExpressionSearch];
         
+        BOOL descriptionError = descriptionString.length == 0;
         BOOL hostnameError = ( !hostname || (hostnameRange.location == NSNotFound) );
         BOOL portIsInvalid = ([port rangeOfString:@"^[0-9]*$" options:NSRegularExpressionSearch].location == NSNotFound);
         BOOL usernameError = username.length == 0;
         BOOL passwordError = password.length == 0;
         BOOL serviceDocError = serviceDoc.length == 0;
         
-        didChangeAndIsValid = !hostnameError && !portIsInvalid && !usernameError && !passwordError && !serviceDocError;
+        didChangeAndIsValid = !descriptionError && !hostnameError && !portIsInvalid && !usernameError && !passwordError && !serviceDocError;
         
         if (didChangeAndIsValid)
         {
+            if ([self.formBackupAccount.accountDescription isEqualToString:self.descriptionTextField.text] == NO)
```

Comment:
Minor: try to follow existing coding styles. For example here we don't use `== NO` but `!=` instead